experimental/bazel-base/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update && \
         python-dev \
         zip \
         unzip \
+        uuid-dev \
```

Comment:
is it possible to build these with Bazel? is it this one? https://github.com/google/uuid

Reply:
This particular package is a linux optional one: https://en.wikipedia.org/wiki/Util-linux

Reply:
ok it's probably not that one :D

Reply:
also i'm surprpsied we don't need pcap here